README.md

```diff
@@ -97,22 +108,29 @@ people shouldn't have indefinite access to, like a share link.
 => #<Document:0x007fae94bf6298 @id="5">
 ```
 
-Note that an explicit `:expires_at` takes precedence over a relative `:expires_in`.
+It's also possible to pass a specific expiry time
 
-You can assign a default SGID lifetime like so:
+```
+>> explicit_expiring_sgid = SecretAgentMessage.find(5).to_sgid(expires_at: Time.now.advance(hours: 1))
+=> #<SignedGlobalID:0x008fde45df8937 ...>
 
-```ruby
-SignedGlobalID.expires_in = 1.month
+# 1 hour later...
+>> GlobalID::Locator.locate_signed explicit_expiring_sgid.to_s
+=> nil
 ```
+Note that an explicit `:expires_at` takes precedence over a relative `:expires_in`.
 
-This way any generated SGID will use that relative expiry.
+**Labels**
```

Comment:
This should be `Purpose` since that's the language GlobalID uses internally. I'll fix that up manually.